training/README.md

```diff
@@ -5,10 +5,11 @@ In order to run accelerated AI workloads, we've prepared [bootc](https://github.
 
 # Makefile targets
 
-| Target     | Description                                   |
-|------------|-----------------------------------------------|
-| amd        | Create bootable container for AMD platform    |
-| nvidia     | Create bootable container for NVidia platform |
+| Target      | Description                                             |
+|-------------|---------------------------------------------------------|
+| amd         | Create bootable container for AMD platform              |
+| nvidia      | Create bootable container for NVidia platform           |
+| intel-gaudi | Create bootable container for Intel Habanalabs platform |
```

Comment:
Do you think there will be other intel-nogaudi repos?  I would prefer to stick with just intel.

Reply:
I'm not sure about the branding to be honest. I know there are some consumer Intel GPUs, but I don't know if those will ever be used for AI or not

Reply:
I think the same could be said for nvidia and amd GPUs.  I think we should stick to intel, and then if we add some other intel gpus that we want to support but have to do it with other images, then we add intel-OTHERGPU.  With a strong preference for not doing this, and having Intel support all of their GPUs from a RHEL perspective.

Reply:
Yes

Reply:
Done

Reply:
intel will have a oneapi variant which is unrelated to this stack, it won't make sense to make a container with both gaudi and oneAPI supported devices (like discrete GPUs).